### PR TITLE
Update the unspecified IP blocking documentation

### PR DESCRIPTION
**Pull request by farcaller**
**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/contributing/).

---
**What does this PR aim to accomplish?:**
Update the documentation reflecting 0.0.0.0 being treated like 127.0.0.1 in some cases. This behavior can trigger in a bunch of networking stacks, e.g. Zoom on macOS will try to connect to a local webserver if some zoom urls are blocked. Similarly, most of Synology's software running on Linux will try to do the same.

---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

### CODE REVIEW

File: docs/ftldns/blockingmode.md [hunk 1]
```diff
@@ -27,14 +27,13 @@ Following [RFC 3513, Internet Protocol Version 6 (IPv6) Addressing Architecture,
...
 
-- The client does not even try to establish a connection for the requested website
 - Speedup and less traffic
-- Solves potential HTTPS timeouts as requests are never performed
 - No need to run a web server on your Pi-hole (reduces complexity when running other web services on the same machine)
 
 ### Disadvantage
 
 - Blocking page cannot be shown and whitelisting has to be performed from the dashboard or CLI
+- Some networking stacks and applications might treat 0.0.0.0 as if it was 127.0.0.1 and try to connect to a local webserver if one's running; causing various connectivity issues
```
Comment (DL6ER) [inline docs/ftldns/blockingmode.md:36]:
```suggestion
- Some networking stacks and applications might treat `0.0.0.0` as if it was `127.0.0.1` and try to connect to a local webserver if one's running; possibly causing connectivity issues
```

File: docs/ftldns/blockingmode.md [hunk 2]
```diff
@@ -27,14 +27,13 @@ Following [RFC 3513, Internet Protocol Version 6 (IPv6) Addressing Architecture,
 
 ### Advantages
 
-- The client does not even try to establish a connection for the requested website
 - Speedup and less traffic
-- Solves potential HTTPS timeouts as requests are never performed
 - No need to run a web server on your Pi-hole (reduces complexity when running other web services on the same machine)
```
Comment (DL6ER) [inline docs/ftldns/blockingmode.md:31]:
```suggestion
- Solves potential HTTPS timeouts as requests may never be performed (depending on your system)
- No need to run a web server on your Pi-hole (reduces complexity when running other web services on the same machine)
```

**Review by DL6ER — CHANGES_REQUESTED** [review]
Sorry for the long delay in reviewing. I made two small change suggestions, what do you think? NULL still seems to be the best "universal" solution we have at hand, we shouldn't be too negative about it

**Comment by farcaller** [comment]
> Sorry for the long delay in reviewing. I made two small change suggestions, what do you think? NULL still seems to be the best "universal" solution we have at hand, we shouldn't be too negative about it

Yeah, with a caveat of "depends on your OS" 😄

**Comment by farcaller** [comment]
derp, it's not like I get email notifications for labels...

**Comment by farcaller** [comment]
...but it looks like this was already merged in with a similar wording either way, so no point in rebasing.

**Pull request closed without merging**